src/utils/statusSeeker.js

```diff
@@ -0,0 +1,16 @@
+import StatusSeekerContract from '../../build/contracts/StatusSeeker.json'
+import contract from 'truffle-contract'
+
+export default {
+  withProvider: (provider) => {
+    const statusSeeker = contract(StatusSeekerContract)
+    statusSeeker.setProvider(provider)
+
+    // @TODO - make this more easily configurable
```

Comment:
Oh, also, any chance that we could make sure that the pluralisation is correct, so that it reads "You have found 1 word" and then "You have found 2 words" etc, rather than the current "1 words" it is returning? 😉

Reply:
You could just commit the /build folder, then everyone will have the correct configuration from when you deployed it to whatever network.

Reply:
I'll quick add this, https://www.npmjs.com/package/pluralize

Otherwise, redux fixed the duplicate issue, and I changed it back to Ropsten because I actually wanted it to work ;)